modules/ztpfw-pipelines-flags-arguments.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: REFERENCE
+[id="ztp-factory-install-pipelines-flags-arguments_{context}"]
+= ZTP factory install pipelines flags and arguments
+
+The pipeline arguments and flags are described in the following tables.k
+
+.Pipeline flags
+[cols="30%,70%",options="header"]
+|===
+| Flag | Description
+
+| `-n`
+|{product-title} namespace where the resources is located.
+
+| `-p`
+| Pipeline parameter.
+
+| `--timeout`
+|Pipeline general timeout.
+
+| `--use-param-defaults`
+| Specifies that apart from the parameters provided, the remainder use the default options.
+
+| `-w`
+|The workspace parameter sets where {product-title} pipelines hold the files during every step. Do not use EmptyDir. The best choice is `name=ztp,claimName=ztp-pvc`. The persistent volume claim  is created during the `bootstrap.sh` execution. It does not need more than 5Gb).
+|===
+
+
+.Pipeline arguments
+[cols="30%,50%,20",options="header"]
+|===
+| Flag | Description | Required
+
+| `Namespace`
+|This is a namespace where all the Tasks and Pipelines will be deployed.
+|Yes
+
+| `git-revision`
```

Comment:
We can skip this describing this option IIUC as the git repo is now a part of the used image.
/cc @jparrill